packages/vitest/src/node/stdin.ts

```diff
@@ -12,8 +12,8 @@ const keys = [
 export function printShortcutsHelp() {
   process.stdout.write(
     `
-${c.bold('Watch Usage')}
-${keys.map(i => c.dim('  press ') + c.reset(i[0]) + c.dim(` to ${i[1]}`)).join('\n')}
+${c.bold('  Watch Usage')}
+${keys.map(i => c.dim('  press ') + c.bold(c.reset(i[0])) + c.dim(` to ${i[1]}`)).join('\n')}
```

Comment:
`c.bold(c.reset(i[0]))` doesn't seem to bold words, maybe we should use `c.reset(c.bold(i[0]))`.  

I did the following test
![image](https://user-images.githubusercontent.com/24516654/151089451-214cf3bc-2103-4ac8-a53a-4f9b5bff4cbe.png)